clang/test/CodeGenCXX/cheri/frame_layout_floating_point.cpp

```diff
@@ -1,3 +1,4 @@
+// REQUIRES: mips-registered-target
 // RUN: %cheri_purecap_cc1 -mframe-pointer=all -target-cpu mips4 -msoft-float -mfloat-abi soft  -target-linker-version 305 \
```

Comment:
Also a terrible tests, no idea what it's testing. Can probably delete it.

Reply:
Done.